packages/testsuite/cypress/support/verification-utils.ts

```diff
@@ -71,6 +71,27 @@ Cypress.Commands.add("verifyAttributeAsExpression", (managementEndpoint, address
   });
 });
 
+Cypress.Commands.add("verifyErrorMessage", (errorMessageText) => {
+  cy.get("div.alert.alert-danger.margin-top-large > span:nth-child(2)").should("have.text", errorMessageText);
+});
+
+Cypress.Commands.add("verifyUserName", (expectedUserName) => {
+  cy.get("#header-username").should("have.text", expectedUserName);
+});
+
+Cypress.Commands.add("verifyUserRole", (expectedRoleName) => {
+  cy.get(".active-roles").should(($roles) => {
+    expect($roles.attr(`title`)).contains(expectedRoleName);
+  });
+});
+
+Cypress.Commands.add("verifyNotLoggedIn", (ssoServer) => {
```

Comment:
This is specific to Keycloak. I suggest to keep this method in the test classes/specs.